src/api/app/models/event/base.rb

```diff
@@ -9,6 +9,17 @@ class Base < ApplicationRecord
 
     before_save :shorten_payload_if_necessary
 
+    EXPLANATION_FOR_NOTIFICATIONS =  {
+      'Event::BuildFail'          => 'Receive notifications of build failures for packages which you are involved in',
```

Comment:
Receive notifications of build failures for packages which you are involved in **as**